### PR TITLE
update `fuels` version

### PR DESCRIPTION
**Pull request by nedsalk**
```
LICENSE:
The Fuel Support team submits this PR to help you use the fuel SDKs. 
It is provided "as is" and without warranty for any purpose. 
We are not participants in the project at hand. 
All the rights of this code are reserved to the authors of this repository.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "@tanstack/react-query": "^5.50.1",
     "clsx": "^2.1.1",
     "firebase": "^10.12.4",
-    "fuels": "^0.94.9",
+    "fuels": "^0.95.0",
     "graphql": "^16.9.0",
     "graphql-request": "^7.1.0",
     "mira-dex-ts": "^1.0.15",
```
Comment (petertonysmith94) [inline package.json:20]:
```suggestion
    "mira-dex-ts": "^1.0.21",
```

File: package.json
Comment (petertonysmith94) [inline package.json:1]: We are awaiting confirmation of the `mira-faucet-ts` package upgrade before merging

File: package.json (hunk 1 above)
Comment (nedsalk) [reply package.json:20]: Also update `@fuels/connectors` and `@fuels/react` to `0.32.0`.

**Comment by mpoplavkov** [comment]
Duplicated by https://github.com/mira-amm/mira-amm-web/pull/62

**Comment by petertonysmith94** [comment]
Thanks @mpoplavkov - forgot that one.

**Pull request closed without merging**